src/main/java/com/michelin/kafkactl/service/ResourceService.java

```diff
@@ -180,25 +181,26 @@ public HttpResponse<Resource> apply(ApiResource apiResource, String namespace, R
      * @param apiResource The resource type
      * @param namespace   The namespace
      * @param resource    The resource
+     * @param version     The version of the resource
      * @param dryRun      Is dry run mode ?
      * @param commandSpec The command that triggered the action
      * @return true if deletion succeeded, false otherwise
      */
-    public boolean delete(ApiResource apiResource, String namespace, String resource, boolean dryRun,
-                          CommandSpec commandSpec) {
+    public boolean delete(ApiResource apiResource, String namespace, String resource, @Nullable String version,
+                          boolean dryRun, CommandSpec commandSpec) {
         try {
             HttpResponse<Void> response = apiResource.isNamespaced()
                 ? namespacedClient.delete(namespace, apiResource.getPath(), resource, loginService.getAuthorization(),
-                dryRun)
+                version, dryRun)
                 : nonNamespacedClient.delete(loginService.getAuthorization(), apiResource.getPath(), resource, dryRun);
 
             // Micronaut does not throw exception on 404, so produce a 404 manually
             if (response.getStatus().equals(HttpStatus.NOT_FOUND)) {
                 throw new HttpClientResponseException(response.reason(), response);
             }
 
-            commandSpec.commandLine().getOut().println(formatService.prettifyKind(apiResource.getKind())
-                + " \"" + resource + "\" deleted.");
+            commandSpec.commandLine().getOut().println((version != null ? "Specified version of " : "")
```

Comment:
`Schema "mySchema" version 3 deleted.` is more elegant, what do you think?

Reply:
It is, I will update the message